community/kernel/src/test/java/org/neo4j/kernel/impl/core/TestLoopRelationships.java

```diff
@@ -71,11 +71,11 @@ public void canDeleteNodeAfterDeletingItsRelationshipsIfThoseRelationshipsInclud
         txCreateRel( node );
         txCreateLoop( node );
 
-        node.delete();
         for ( Relationship rel : node.getRelationships() )
         {
             rel.delete();
         }
+        node.delete();
```

Comment:
If this change is required then this means there's a user-facing behaviour change which might break a lot of code out there. Is there a good reason to make this behaviour change?